### ISSUE TITLE
Method Illuminate\Auth\RequestGuard::getUser does not exist.

### ISSUE THREAD

**Issue by cmdobueno** [laravel/telescope#28]
Alright I have installed telescope on an existing laravel 5.7 application.

This application is a vue spa that uses passport for authentication. 

```
Method Illuminate\Auth\RequestGuard::getUser does not exist.
```
This is the error I receive, I have attempted to update the Authorize class uses in the config file, but I have had very little luck with it as of yet... Anybody advice or suggestions would greatly be appreciated.

**Comment by ForzaSFerrari** [comment]
Have the same error

**Comment by cmdobueno** [comment]
There is a 'fix'.

@line 196 of Telescope.php
```
if( Auth::hasUser() ) {
			try {
				$entry->user( Auth::getUser() );
			}
			catch( Exception $e ) {
				$entry->user( Auth::user() );
			}
		}
```

I am not good with github, but this fixed my issue...

**Comment by themsaid** [comment]
fixed

**Comment by daniel-de-wit** [comment]
For reference, fixed in commit: fa6919367e9d0af7bbac05c0b3fa89707239010d
